### PR TITLE
[1LP][RFR] Replacing all the None getters

### PR DESCRIPTION
**Pull request by psav**
* Making Quantified Code happy

```find -type f -name "*.py" | xargs sed -i 's/\(.get(.*\), None/\1/'```

{{pytest: cfme/tests/configure/test_ntp_server.py}}

### CODE REVIEW

File: cfme/web_ui/utilization.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def list_data_chart(self, raw=False):
                         datetime.strptime(date_format.format(_date), "%m/%d/%Y %H:%M:%S %Z"))
                 _data = {time_format: _date}
                 # ignore duplicate values for timestamp
-                if not next((item for item in data if item.get(time_format) == _date), None):
```
Comment (psav) [inline cfme/web_ui/utilization.py]: Mistake

File: utils/appliance/__init__.py [hunk 2]
```diff
@@ -1662,7 +1662,7 @@ def log_raise(exception_class, message):
             raise exception_class(message)
 
         if vddk_url is None:  # fallback to VDDK 5.5
-            vddk_url = conf.cfme_data.get("basic_info", {}).get("vddk_url", None).get("v5_5", None)
+            vddk_url = conf.cfme_data.get("basic_info", {}).get("vddk_url", None).get("v5_5")
```
Comment (psav) [inline utils/appliance/__init__.py]: Potential error here

File: cfme/tests/configure/test_log_depot_operation.py [hunk 3]
```diff
@@ -84,7 +84,7 @@ def pytest_generate_tests(metafunc):
         if proto_data['use_for_log_collection']:
             depots.append([LogDepotType(
                 protocol, creds,
-                proto_data.get('sub_folder', None), proto_data.get('path_on_host', None))])
```
Comment (mshriver) [inline cfme/tests/configure/test_log_depot_operation.py]: Missed one here?

File: scripts/clone_template.py [hunk 4]
```diff
@@ -114,26 +114,26 @@ def main(**kwargs):
...
 
     # Try to snag defaults from cfme_data here for each provider type
     if provider_type == 'rhevm':
-        cluster = provider_dict.get('default_cluster', kwargs.get('cluster', None))
+        cluster = provider_dict.get('default_cluster', kwargs.get('cluster'))
         if cluster is None:
             raise Exception('--cluster is required for rhev instances and default is not set')
         deploy_args['cluster'] = cluster
 
-        if kwargs.get('place_policy_host', None) and kwargs.get('place_policy_aff', None):
```
Comment (mshriver) [inline scripts/clone_template.py]: missed one here?

File: scripts/clone_template.py [hunk 5]
```diff
@@ -225,7 +225,7 @@ def main(**kwargs):
             ssh_client.close()
         return 10
 
-    if kwargs.get('outfile', None) or kwargs.get('deploy', None):
```
Comment (mshriver) [inline scripts/clone_template.py]: missed one

File: utils/appliance/__init__.py [hunk 6]
```diff
@@ -1662,7 +1662,7 @@ def log_raise(exception_class, message):
             raise exception_class(message)
 
         if vddk_url is None:  # fallback to VDDK 5.5
-            vddk_url = conf.cfme_data.get("basic_info", {}).get("vddk_url", None).get("v5_5", None)
+            vddk_url = conf.cfme_data.get("basic_info", {}).get("vddk_url", {}).get("v5_5")
```
Comment (mshriver) [inline utils/appliance/__init__.py]: :+1:

**Comment by mshriver** [comment]
@psav marked WIPTEST, I'll merge once its back to RFR

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**